addons/account/__init__.py

```diff
@@ -16,7 +16,7 @@ def _auto_install_l10n(cr, registry):
     country_code = env.company.country_id.code
     if country_code:
         #auto install localization module(s) if available
-        to_install_l10n = env['ir.module.module'].search_count([('name', 'like', 'l10n_'), ('state', '=', 'to install')])
+        to_install_l10n = env['ir.module.module'].search_count([('name', '=like', 'l10n\___'), ('state', '=', 'to install')])
```

Comment:
To consider the module `l10n_generic_coa`, a domain like te following would work IMHO:

```
[
    ('state', '=', 'to install'),
    '|',
    ('name', '=', 'l10n_generic_coa'),
    ('name', '=like', 'l10n\___'),
]
```

Reply:
l10n_syscohada